WebHostLib/templates/stats.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'pageWrapper.html' %}
+
+{% block head %}
+    <title>Archipelago Game Statistics</title>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename="styles/markdown.css") }}" />
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename="styles/stats.css") }}" />
+    {{ css_resources|indent(4)|safe }}
+    {{ js_resources|indent(4)|safe }}
+    {{ chart_data|indent(4)|safe }}
+{% endblock %}
+
+{% block body %}
+    {% include 'header/grassFlowersHeader.html' %}
+    <div id="stats" class="markdown">
+        <h1>Archipelago Game Statistics</h1>
+        <h5>
+            The data on this page is updated hourly.
+        </h5>
+
+        <div id="charts-wrapper">
+        {% for chart in charts %}
+            <div class="chart-container">
+                {{ chart|indent(12)|safe }}
```

Comment:
indent(12) is now no longer accurate indentation. If we don't care about what the raw html looks like we can just drop the filter, or if we do we can update it to 16.

Reply:
I've updated the chart indent, but I'd actually like the designer to take a look at the charts before I try to style them myself. It doesn't look bad, so it's probably save to deploy like this.